tools/c7n_gcp/c7n_gcp/resources/iam.py

```diff
@@ -174,6 +180,10 @@ class resource_type(TypeInfo):
         name = id = "name"
         default_report_fields = ['name', 'title', 'description', 'stage', 'deleted']
         asset_type = "iam.googleapis.com/Role"
+        urn_component = "role"
+        # Don't show the project ID in the URN.
```

Comment:
what's the rationale on this?

Reply:
because they are global roles... I don't think there are any project identifiers in the self link. compare with project-roles.

Reply:
are they globaly namespaced? how do we ensure uniquness of the urn across projects?